Terasology/CombatSystem
# feat: add slimeBall projectile and clean up projectile assets

- rename basicProjectile -> basicShootWithGravity to make clearer what it is
- align references with renaming
- remove duplicated Components that are already present in parent prefabs
- add slimeBall projectile
- add PlasmaShot sound

Extracted from [GooKeeper](https://github.com/Terasology/GooKeeper) module in the context of https://github.com/Terasology/GooKeeper/issues/59

Depends on https://github.com/MovingBlocks/Terasology/pull/5000

### How to test

Reference `CombatSystem:slimeBall` as `launchEntityPrefab` in `assets/prefabs/items/staff.prefab`.
Start the game and `give staff`. Right-click to see the slimeball projectile.
